BraveWallet/Crypto/Transaction Confirmations/TransactionConfirmationView.swift

```diff
@@ -49,28 +42,28 @@ struct TransactionConfirmationView: View {
   }
 
   private var transactionDetails: String {
```

Comment:
I wonder if it would be better to parse this in the `TransactionConfirmationStore` and add a new property to the `State` struct?